service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
+
+echo "setting up test environment"
+
+# Set the working directory to the script's directory
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd -P)
+cd $script_dir
+echo "Generate busybox rootfs directory for runC tests"
+rootfs=runtime/runc/testbundle/rootfs
+mkdir -p $rootfs
+sudo mkdir -p /var/run/gcsrunc
+sudo docker export $(sudo docker create busybox) | tar -C $rootfs -xvf - > /dev/null
+
+echo "clone opencontailers/runc repo and build runc binary"
+
+cd $GOPATH/src/github.com
+mkdir opencontainers && cd opencontainers
+git clone https://github.com/opencontainers/runc runc
+cd runc && git checkout 992a5be178a62e026f4069f443c6164912adbf09
```

Comment:
Well, the point of @jterry75's recent change (https://github.com/Microsoft/opengcs/pull/58) is not to rely on the runc executable being in any particular location, as long as it's in the path. So I don't think it necessarily makes sense to assume runc will be put in /bin or /sbin or wherever.